content/userinfo/storage.md

```diff
@@ -89,9 +89,13 @@ aliases = [ "/storage" ]
 
 {{< storage_main_page_second >}}
 
-Researchers who request space on ICS must first request an Ivy account using the [Ivy request form](https://services.rc.virginia.edu/ivyvm). Further information on Ivy and the ICS can be found [here](/userinfo/storage/sensitive-data/#ivy-central-storage).
+Researchers who request space on High-Security Research Standard must first request an Ivy account using the [Ivy request form](https://services.rc.virginia.edu/ivyvm). Further information on Ivy and the High-Security Research storage can be found [here](/userinfo/storage/sensitive-data/#hs-standard-storage).
```

Comment:
Small change, add "Standard" to "Further information on Ivy and the High-Security Research Standard storage"